adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapter.java

```diff
@@ -354,7 +369,7 @@ private void handleSessionOpen(final ProtonConnection conn, final ProtonSession
      * @param con The connection to close.
      * @param res The client's close frame.
      */
-    private void handleRemoteConnectionClose(final ProtonConnection con, final AsyncResult<ProtonConnection> res) {
+    private void handleRemoteConnectionClose(final ProtonConnection con, final AsyncResult<ProtonConnection> res, final Device device) {
```

Comment:
there doesn't seem to be a need for the device param

Reply:
I will remove it.